src/main/java/com/elastic/support/diagnostics/commands/VersionCheckCmd.java

```diff
@@ -31,14 +33,19 @@ public void execute(DiagnosticContext context) {
         try {
             DiagnosticInputs diagnosticInputs = context.getDiagnosticInputs();
             RestClient restClient = context.getEsRestClient();
+            RestResult res = restClient.execQuery("/");
+            if(res.getStatus() != 200)){
+                throw new DiagnosticException("Unrecoverable ES query error.");
```

Comment:
```suggestion
                throw new DiagnosticException("Unable to fetch [GET /]");
```

Since this is in the `try` / `catch`, this info will go well with the error output.